openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js

```diff
@@ -0,0 +1,29 @@
+/*
+  This file is to enable users on the coursetalk reviews
```

Comment:
Use all 79 characters before a newline.  Also, I'd mention switching between the two view.  Maybe something like:
```
Enable users to switch between viewing and writing CourseTalk reviews.
```